docs/development/standards.rst

```diff
@@ -73,6 +70,12 @@ After cloning ``readthedocs.org`` repository, you need to
 
 #. install `Docker <https://www.docker.com/>`_ following `their installation guide <https://docs.docker.com/install/>`_.
 
+#. clone the ``readthedocs.org`` repository:
+
+   .. prompt:: bash
+
+      git clone --recurse-submodules git@github.com:readthedocs/readthedocs.org.git
```

Comment:
Using a https url is more easy for users since they don't need to setup a ssh key